lib/main.dart

```diff
@@ -1,4 +1,6 @@
 import 'package:flutter/material.dart';
+import 'screens/profile_page.dart';
+import 'package:profile_page/widget/my_profile.dart';
```

Comment:
the widget you're trying to import doesn't exists in the file you submitted could be that you forgot to push it in git hub